index.html

```diff
@@ -38,6 +38,193 @@ <h2 class="sub-tittle">I’am a Software Developer</h2>
           </ul> 
         </div>
       </section>
+      <!--  -->
+<section class="second-sec">
+  <h3 class="second-tittle">
+    My Recent Works
+  </h3>
+  <img width="327" height="237" src="./images/Img-Placeholder.png" alt="">
+  <h3 class="second-subtittle">
+    Multi-Post Stories
+  </h3>
```

Comment:
### **Great implementation of the project requirements**

- [ ] Please your design misses the horizontal line under the Recent Work title and your font doesn't match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) font.
### **Take Look at Figmal Design**
![figmalRecentWork](https://user-images.githubusercontent.com/55596970/185563824-b0f3ceba-f8d0-4c08-b1b6-c6ac6484c2ec.png)
### **Your Design**
![yourfont](https://user-images.githubusercontent.com/55596970/185563998-37e6fbc9-5878-481e-ba08-5406bff3537f.png)